frontend/ui/widget/frontlightwidget.lua

```diff
@@ -48,7 +52,7 @@ function FrontLightWidget:init()
     self.fl_max = self.powerd.fl_max
     self.fl_cur = self.powerd:frontlightIntensity()
     local steps_fl = self.fl_max - self.fl_min + 1
-    self.one_step = math.ceil(steps_fl / 25)
+    self.one_step = math.ceil(steps_fl / 100)
```

Comment:
Accidental leftover from a quick test auto-generating loads of buttons, but it's a lucky coincidence because it draws attention to the fact that this whole one_step thing isn't even used in this concept and should be removed.